time/README.md

```diff
@@ -52,14 +52,16 @@ Output:
 
 Normally, the Blocky AS runtime does not provide sandboxed functions access to
 the system clock. Instead, they are given a hardcoded time that monotonically 
-increases by "1ns" for every subsequent call. Starting with 0.1.0-beta.6, the
-Blocky AS runtime fetches the current time from a remote time-server and
-provides these values to functions in the sandboxed environment via standard
-system calls.
-
-This means that your Blocky AS functions can fetch time using standard library
-calls same as if they were executing in a non-TEE environment. We define a 
-simple function in [`main.go`](./main.go) that calls `time.Now()`.
+increases by "1ns" for every subsequent call.
+
+Starting with 0.1.0-beta.9, guest functions have access
+to time from [`PTP`](https://en.wikipedia.org/wiki/Precision_Time_Protocol).
+Each time a standard library function, like `time.Now()` is invoked the system
+call results in Blocky AS runtime fetching time from `/dev/ptp0` device.
```

Comment:
```suggestion
Each time a guest function calls a standard library time function, like `time.Now()`,  the resulting time system
call triggers the Blocky AS runtime to fetch current time from the `/dev/ptp0` device.
```

Reply:
I think my suggestion will blow past the line character limit - could you adjust?